src/urllib3/exceptions.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import
-from .packages.six.moves.http_client import IncompleteRead as httplib_IncompleteRead
```

Comment:
I'm guessing this `# type: ignore` is temporary? Should we start by typing `urllib3.packages.six` per the dependency tree?